test/cookbooks/rundeck-test/recipes/default.rb

```diff
@@ -1,5 +1,7 @@
 include_recipe 'rundeck-server'
 
+package 'curl'
```

Comment:
Because a couple existing `rundeck_server_job` resources later in the recipe have guards using the curl command, and the Ubuntu bento boxes don't (necessarily) have curl installed by default.